docs/production/production.md

```diff
@@ -0,0 +1,11 @@
+## Production
```

Comment:
can we also link to ours, also we should use the production release binaries in the repository, also where is our custom spec file?

Reply:
why would we need to link to our chainspecs?